osu.Game/Graphics/UserInterface/OsuAnimatedButton.cs

```diff
@@ -122,7 +122,7 @@ protected override bool OnClick(ClickEvent e)
         protected override bool OnMouseDown(MouseDownEvent e)
         {
             Content.ScaleTo(ScaleOnMouseDown, 2000, Easing.OutQuint);
-            return base.OnMouseDown(e);
+            return true;
```

Comment:
tangential and I expect no action to be taken, but every time changes like these show up I expect some *very* stupid breakage to occur. that said tests pass and I'm not about to spend an hour looking through every inheritor of `OsuAnimatedButton` to look for breakage, so I'm probably going to let this be and let any breakage (if any) occur

Reply:
I ran through tests and checked a few places I expected to break and all looked okay, but I completely agree.

Reply:
This case is annoying as fuck. Makes me really concerned about how context menus are implemented.